src/stormd/service.rs

```diff
@@ -197,11 +199,23 @@ impl Runtime {
 
                 let (container_id, instr) = match mesg {
                     // These should be processed by transfer service
-                    // TODO: Ensure that the incoming chunks references correct app id and message
```

Comment:
This todo is not resolved here; its real meaning is that we need to ensure that the app has the access to the chunks, i.e. they were created by the same app originally. Since this PR is not related to that we need to leave TODO here after the bugfix.

Reply:
Ok.